src/presentation/controllers/configuration-controller.ts

```diff
@@ -0,0 +1,44 @@
+import { BaseController, GetRequest, RestController } from './base-controller'
+import { HttpErrorHandler } from '../interfaces/http-error-handler'
+import { Request, Response } from 'express'
+import { ConfigurationRepository } from '../../data-access/repositories/interfaces/configuration-repository'
+import { Exception } from '../../model/exceptions/exception'
+import { Configuration } from '../../model/entities/configuration'
+import { ShowStyleVariantRepository } from '../../data-access/repositories/interfaces/show-style-variant-repository'
+import { ShowStyleVariant } from '../../model/entities/show-style-variant'
+
+@RestController('/configurations')
+export class ConfigurationController extends BaseController {
+  constructor(
+    private readonly configurationRepository: ConfigurationRepository,
+    private readonly showStyleVariantRepository: ShowStyleVariantRepository,
+    private readonly httpErrorHandler: HttpErrorHandler
+  ) {
+    super()
+  }
+
+  @GetRequest()
+  public async getBlueprintConfiguration(_req: Request, res: Response): Promise<void> {
+    try {
+      const configuration: Configuration = await this.configurationRepository.getConfiguration()
+      res.send({
+        showStyleConfiguration: configuration.showStyle.blueprintConfiguration,
+        studioConfiguration: configuration.studio.blueprintConfiguration
+      })
```

Comment:
There is a slight mismatch between the URL and what the endpoint returns.
The URL is `/configurations` so I would expect to get a Configuration for the entire system, but what I am receving is just the BlueprintsConfigurations (which fit with the method name).

Either the URL should be changed to be `/configurations/blueprints` or the method should be updated to return the entire Configuration `res.send(configuration)`

Reply:
Not to mention that we don't have an id for which configuration we want to return so `/configurations` should by REST standards return a list of all Configurations.
Not sure if we should change that now...

Reply:
Hmm, i don't know if it would be usefull for client to know of the layers in `Studio`. 
For know i will change the endpoint to include `/blueprints`.